bot/exts/info/tags.py

```diff
@@ -138,6 +140,15 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.tags: dict[TagIdentifier, Tag] = {}
         self.initialize_tags()
+        self.bot.tree.copy_global_to(guild=discord.Object(id=GUILD_ID))
+
+    tag_group = app_commands.Group(name="tag", description="...")
+    # search_tag = app_commands.Group(name="search", description="...", parent=tag_group)
+
+    @Cog.listener()
+    async def on_ready(self) -> None:
+        """Called when the cog is ready."""
+        await self.bot.tree.sync(guild=discord.Object(id=GUILD_ID))
```

Comment:
I don't think any of the syncing code should be necessary, since we do that automatically now (https://github.com/python-discord/bot-core/pull/171)

Reply:
As a general advice, **nothing** should be done in the `on_ready` event listener, as it could get triggered multiple times, and syncing is an expensive task.

Instead, you should use the `cog_load` hook for things that need to happen once post instantiation of the cog.

2. This also hasn't been resolved it seems